kyo-core/shared/src/main/scala/kyo/Meter.scala

```diff
@@ -76,24 +83,36 @@ object Meter:
         def closed(using Frame): Boolean < IO      = false
     end Noop
 
-    /** Creates a Meter that acts as a mutex (binary semaphore).
+    /** Creates a **reentrant** Meter that acts as a mutex (binary semaphore).
       *
       * @return
       *   A Meter effect that represents a mutex.
       */
     def initMutex(using Frame): Meter < IO =
```

Comment:
Why not use default param? worse binary compat?

Reply:
With a default param, it's not possible to call `Meter.initMutex` without an empty `()` at the end.